packages/squiggle-lang/src/rescript/Distributions/PointSetDist/AlgebraicShapeCombination.res

```diff
@@ -251,3 +254,5 @@ let combineShapesContinuousDiscrete = (
     XYShape.T.empty,
   )
 }
+
+let checkOrdered = (a: XYShape.T.t): bool => a.xs[0] < a.xs[1]
```

Comment:
Two issues:
1. I think this should be called ``isOrdered``
2. Why are you only testing the first two points?

Reply:
Fixed both. I had a justification for the second, but on second thought I don't like it